### PR TITLE
Add provider policies and update candidate policies

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
### Context

* Splits out footer from application layout to show different support text and provider different links to policies, based on whether candidate, provider or support interface is being shown.
* There is now a single privacy policy that applies to all users
* Adds terms of use for providers
* Adds cookie policy for providers (currently uses that same as that for candidates, but replaces ‘Apply for teacher training’ with ‘Manage teacher training applications’)
* Adds four new routes (that share the same Markdown content behind the scenes)
  * `/candidate/accessibility`
  * `/canddiate/privacy-policy`
  * `/provider/accessibility`
  * `/provider/privacy-policy`
* Fixes incorrect page titles appearing in `<title>` for rendered content pages

In terms of code:
* Provider and candidate interfaces each have their own `ContentController`
* `render_content_page` has been moved into a shared `ContentHelper`
* Existing content page tests moved to `/spec/system/candidate_interface`
* Adds tests for provider content to `/spec/system/provider_interface`

### Changes proposed in this pull request

Footer variations:

#### Candidate

Screenshot 2019-11-20 at 15 34 21

#### Provider

Screenshot 2019-11-20 at 15 34 34

#### Support

Screenshot 2019-11-20 at 15 35 08

### Link to Trello card

[379 - Privacy statement](https://trello.com/c/VDuA3Ndv)
[1287 - Terms and conditions added to provider interface](https://trello.com/c/xyhzjfrP)
[1288 - Privacy statement added to the provider interface](https://trello.com/c/9dI6NZUT)

### CODE REVIEW

File: app/controllers/provider_interface/content_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module ProviderInterface
+  class ContentController < ActionController::Base
```
Comment (paulrobertlloyd) [inline app/controllers/provider_interface/content_controller.rb]: Would ideally inherit from `ProviderInterfaceController` here, but that forces authentication, and you don’t need to be signed in to view policies. I pulled across the relevant parts of that controller to use here, but open to suggestions if better way to achieve this.

File: app/views/content/terms_provider.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+Thank you for taking part in trialling Manage teacher training applications. Here’s what you can expect from us, and what we ask of you, over the course of the pilot.
+
+## What you can expect from us
+
+1. ### Designated support
+
+    We will have designated support to answer your queries. You can contact us by:
+    * phone: INSERT TELEPHONE NUMBER
```
Comment (paulrobertlloyd) [inline app/views/content/terms_provider.md]: There’s a card in Trello to review these policies before we launch – `INSERT TELEPHONE NUMBER` is one such thing we need to ensure we have in place.

File: app/controllers/provider_interface/content_controller.rb (hunk 1 above)
Comment (tijmenb) [reply app/controllers/provider_interface/content_controller.rb]: You can inherit from `ProviderInterfaceController` and add:

```rb
skip_before_action :authenticate_provider_user!
```

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -91,24 +91,12 @@
...
-              <li class="govuk-footer__inline-list-item">
-                <%= link_to t('layout.cookie_policy'), cookies_candidate_path, class: 'govuk-footer__link' %>
-              </li>
-            </ul>
+            <% case current_namespace %>
+            <% when 'candidate_interface' %>
+              <%= render 'layouts/candidate_footer' %>
+            <% when 'provider_interface' %>
+              <%= render 'layouts/provider_footer' %>
+            <% end %>
```
Comment (tvararu) [inline app/views/layouts/application.html.erb]: 👍

File: app/controllers/provider_interface/content_controller.rb (hunk 1 above)
Comment (paulrobertlloyd) [reply app/controllers/provider_interface/content_controller.rb]: Oooooh 😀

File: app/views/content/cookies.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+Apply for teacher training puts small files (known as ‘cookies’) onto your computer to collect information about how you use the site.
+
+Cookies are used to:
+
+* measure how you use the this service so it can be updated and improved based on your needs
+* remember the notifications you’ve seen so that we don’t show them to you again
+* remember who you are when you have logged in so we can show you your information and information relevant to you
+* ’Apply for teacher training’ cookies aren’t used to identify you personally.
+
+Find out more about [how to manage cookies](http://www.aboutcookies.org/).
```
Comment (fofr) [inline app/views/content/cookies.md]: This feels like a weird thing to link to?

Comment (paulrobertlloyd) [reply app/views/content/cookies.md]: This is content @dankmitchell ported from Find. Probably needs reviewing (and potentially across the service line)

**Pull request merged**